icu4c/source/test/testdata/rbbitst.txt

```diff
@@ -1525,6 +1525,31 @@ Bangkok)•</data>
 #
 ####################################################################################
 
+# -u-dx (exclude script)
+#<locale th
+
+
+#<line>
+
+<locale sss@dx=thai>
+<line>
+# Should no longer break at the dictionary points - it's not Thai language
+# Short Test
+<data>•โอํน• อะไป •จู่วาม •โล่น•</data>
+#<data>•โอํน• •อะไป• •จู่วาม• •โล่น• •เปี่ยร• •อะลู่วาง• •แมะ,• •ปาย• •อัน• •แบ็จ• •อะโจํน• •ซา• •เมาะ.• •อัน• •ฮะบืน• •ตะ• •เวี่ยะ• •ตะ• •งี่ยาน,• •อัน• •ฮะบืน• •อีว• •อะปายฮ.•</data>
+#<word>
+# Should no longer break at the dictionary points - it's not the Thai language
+#<data>•โอํน• •อะไป• •จู่วาม• •โล่น• •เปี่ยร• •อะลู่วาง• •แมะ,• •ปาย• •อัน• •แบ็จ• •อะโจํน• •ซา• •เมาะ.• •อัน• •ฮะบืน• •ตะ• •เวี่ยะ• •ตะ• •งี่ยาน,• •อัน• •ฮะบืน• •อีว• •อะปายฮ.•</data>
+
+#<locale sss@dx=zyyy>
```

Comment:
using dx=zyyy is very troublesome . I filed a CLDR bug in https://unicode-org.atlassian.net/browse/CLDR-17247 for that. From my point of view, that is a very bad way to exclude "all scripts"